examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,30 +61,28 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+X, labels = EpochsVectorizer().fit_transform(epochs)
+clf = make_pipeline(XdawnTransformer(n_components=3,
+                    n_chan=epochs.info['nchan']),
                     MinMaxScaler(),
                     LogisticRegression(penalty='l1'))
 
```

Comment:
node that `n_chan` will have to be removed in the next PR since we'll pass n-d arrays